main.go

```diff
@@ -319,21 +319,21 @@ func connectAction(ctx *cli.Context) error {
 		// Update the configuration file
 		// Call D-bus to get the CA directory from rhsm
 		if err = getRHSMConfigOption("rhsm.ca_cert_dir", &config.CADir); err != nil {
-			errorMessages[BrandName] = fmt.Errorf("cannot get rhsm configuration: %w", err)
+			log.Warn("cannot get rhsm configuration: %w", err)
 		}
 		// Generate a new http client configured with mutual TLS certificate
 		tlsConfig, err := config.CreateTLSClientConfig()
 		if err != nil {
-			errorMessages[BrandName] = fmt.Errorf("cannot configure TLS: %w", err)
+			log.Warn("Cannot configure TLS: ", err)
 		}
 		client, err := http.NewHTTPClient(tlsConfig)
 		if err != nil {
-			errorMessages[BrandName] = fmt.Errorf("cannot configure HTTP client: %w", err)
+			log.Warn("Cannot configure HTTP client: ", err)
 		}
 		// Get the user profile
 		profile, err := getConfProfile(client)
 		if err != nil {
-			errorMessages[BrandName] = fmt.Errorf("Cannot get the user profile: %w", err)
+			log.Error("Cannot get the user profile: ", err)
```

Comment:
When it is possible to get profile and print results, then blue dot and message is printed. When it is not possible to get profile, then there is no red dot and no error/warning message. It is not consistent.

What is wrong about storing error in the map as we do for "rhsm", "insights-client" and "rhcd"? Why is getting profile inconsistent again? I can understand that it is not good to return some error code, when it is not possible to get information about profile, because system was just registered.

Printing all errors in table was probably not the best UX decision (as we learned here: https://github.com/RedHatInsights/rhc/pull/29#issuecomment-1401823780), but we will solve this later.

I can see that it easy to solve this issue with `log.Warn()` and `log.Error()`, but I think that we can do better.

I have proposal. Not store only error message in `errorMessage`, but store there also severity and later in `showErrorMessages()` function you can return `nil`, when only warnings were printed.

Reply:
Also, adding this as an error could be causing automatic playbooks/scripts to fail when `rhc` has correctly registered and `rhcd` is up and running. Only by not getting the profile, this is common situation for stage uses without proper configure `base_url`.

Adding those logs into errorMessage and then investigating inside that to know if the error is critical or not seems a bit over-engineering. I would like to have @subpop minds in here.

Reply:
> Wow I thought this PR was already merged :crying_cat_face:

Blame @subpop . He forgets to push merge button after approving PR :smile_cat:

Reply:
Why do we collect errors in a map to begin with? That seems weird. Can we use a custom logger/printer function that prefixes the string with the correct dot and logs it when the error occurs? Or are we collecting then errors because it breaks the terminal animation?

Reply:
I think that the main reason is terminal animation... When we have some existing design pattern, then we should follow it. Why? Several reasons:

1. It looks better, when all parts of application are consistent.
2. If it decide to change terminal animation in the future, then it will be easier to redo, when all steps use same design pattern.

Reply:
These are two very convincing points. I appreciate it when code follows the existing patterns within the application; it makes for easier long-term maintenance. It seems like perhaps we could store the log level value in the map as well; that would allow for consistent handling of all the errors.

Reply:
I didn't want to blame anyone :laughing: 
Ok, this new approach makes sense for consistence. I'll be working on it (also add this as a jira ticket to track its status)